src/unix/udp.c

```diff
@@ -314,6 +368,22 @@ static void uv__udp_sendmsg(uv_udp_t* handle) {
         abort();
       }
     }
+
+#if defined(__linux__)
```

Comment:
I think this will work only if `handle->flags` contains `UV_UDP_PKTINFO`. 
We could add this check too.

Reply:
Fixed.